server/pbench/bin/pbench-unpack-tarballs

```diff
@@ -69,12 +71,12 @@ test -d $RESULTS || doexit "Bad RESULTS=$RESULTS"
 
 # the link source and destination for this script
 linksrc=TODO
-linkdest=TO-COPY-SOS
+linkdest=TO-INDEX
```

Comment:
With the addition of pbench-dispatch, these should change to TO-UNPACK and UNPACKED resp.